manifests/server.pp

```diff
@@ -49,6 +49,8 @@
   Optional[String[1]] $collect_tag                                              = $prometheus::collect_tag,
   Optional[Integer] $max_open_files                                             = $prometheus::max_open_files,
   Stdlib::Absolutepath $usershell                                               = $prometheus::usershell,
+  Optional[String[1]] $proxy_server                                             = $prometheus::proxy_server,
```

Comment:
what kind of proxy address can be passed? an https URL? If so, can you please update the datatype for that? puppetlabs-stdlib has a few that might fit

Reply:
this parameter and the below `$proxy_type` parameter are passed directly to the `archive` resource in install.pp, which accepts any string for `$proxy_server` and one of 'none', 'http', 'https', or 'ftp' for `$proxy_type`. So this matches those requirements (along with the datatypes of all the similar parameters on the manifests for the exporters)